src/AppFrame.tsx

```diff
@@ -1,15 +1,28 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
 import React, { useCallback, useState } from "react";
 import { useLocation } from "react-router-dom";
 import { Frame, Layout, Navigation, Page, TopBar } from "@shopify/polaris";
 import { HomeMajor } from "@shopify/polaris-icons";
 import { Routes } from "./Routes";
 import { TwitterTimelineEmbed } from "react-twitter-embed";
+import { useTranslation } from "react-i18next";
+import i18next from "i18next";
 
 export function AppFrame() {
+  const { t, i18n } = useTranslation(undefined, { useSuspense: false });
```

Comment:
You can't just use `const { t, i18n } = useTranslation()`

Reply:
translating the Home button (Accueil) won't work without this. It gives an error if you remove useSuspense: false. Suspense is a react feature to show a loading spinner/image when data is loading. Alternatively if you wrap the AppFrame in <Suspense> tags it should work